environment.yml

```diff
@@ -1,15 +1,22 @@
 name: torchmd-net
 channels:
-  - pytorch
-  - rusty1s
+  - raimis
   - conda-forge
 dependencies:
-  - matplotlib
-  - pip
-  - pytorch==1.8.0
-  - pytorch-geometric=1.7.2=py36_torch_1.8.0_cpu
-  - pytorch-lightning==1.3.1
   - ase
   - h5py
+  - matplotlib
+  - pip
+  - python
+  - pytorch==1.10.0
+  # The official PyTorch Geometric package is not compatible with conda-forge,
+  # while the package on conda-forge is broken
+  # (https://github.com/pyg-team/pytorch_geometric/issues/3660).
+  # Until a better solution is found a have custom package to plug the gap.
+  - raimis::pytorch-geometric==2.0.2
+  - pytorch-lightning==1.3.8
+  - setuptools==59.5.0 # Pin due to https://github.com/pytorch/pytorch/pull/69904
   - tqdm
-  - scipy
+  # Dev tools
+  - flake8
+  - pytest
```

Comment:
Don't add testing utilities like `pytest` and `flake8` to the env file. If someone wants to install the env to use torchMD-Net he doesn't want the test packages

Reply:
- The current installation instructions don't even mention that environment file.
- When the things are ready for OpenMM/ACEMD, we will have to make a conda-forge package. So only developers will be installing manually.